jujugui/static/gui/src/app/components/inspector-change-version/_inspector-change-version.scss

```diff
@@ -0,0 +1,24 @@
+.inspector-change-version {
+    @extend %scroll-children;
+
+    &__current {
```

Comment:
My understanding of BEM is that a double underscore denotes an element of the block, while a single underscore denotes a modifier. Are these modifiers of a single element, or distinct elements in their own right?